MarsQASpecFlowProject/Pages/LanguagePage.cs

```diff
@@ -0,0 +1,86 @@
+﻿using MarsQASpecFlowProject.Utilities;
+using OpenQA.Selenium;
+
+
+namespace MarsQASpecFlowProject.Pages
+{
+    public class LanguagePage
+    {
+        // Locators
+        public static By LanguageTab = By.XPath("//a[contains(text(),'Languages')]");
+        public static By AddNewButton = By.XPath("//*[@class='ui teal button ']");
+        public static By LanguageTextbox = By.XPath("//*[@placeholder='Add Language']");
```

Comment:
please change the By to return webelement e.g. public static IWebElement LanguageTextbox = driver.FindElement(By.XPath("//*[@placeholder='Add Language']"));

Reply:
then you will do  in the method you will just need to do this LanguageTextbox.Click();

Reply:
Refactor locators to Web Elements.